www/cobrands/groundcontrol/js/groundcontrol_router.js

```diff
@@ -0,0 +1,38 @@
+(function (FMS, Backbone, _, $) {
+    FMS.appRouter = FMS.appRouter.extend({
+        routes: {
+            '': 'home',
+            'home': 'home',
+            'offline': 'offline',
+            'around': 'around',
+            'search': 'search',
+            'existing': 'existing',
+            'photo': 'photo',
+            'details': 'details',
+            'details_extra': 'details_extra',
+            'submit-start': 'submitStart',
+            'submit': 'submit',
+            'submit-email': 'submitEmail',
+            'submit-name': 'submitName',
+            'submit-name-set-password': 'submitNameSetPassword',
+            'submit-password': 'submitPassword',
+            'submit-set-password': 'submitSetPassword',
+            'save_offline': 'saveOffline',
+            'sent': 'sent',
+            'reports': 'reports',
+            'login': 'login',
+
+            // Cobrand additions
+            'quote': 'quote'
+        },
```

Comment:
Is there a way to add `quote` to the routes list rather than having to replicate it all? I think you could call `FMS.appRouter.route()` perhaps? "Manually create a route for the router".